lib/smb2/file.rb

```diff
@@ -4,29 +4,43 @@
 # An open file on the remote filesystem.
 class Smb2::File
 
-  # The tree that {Tree#create created} this File
-  #
-  # @return [Smb2::Tree]
-  attr_accessor :tree
+  # Some day we'll document all of these in one place
+  STATUS_END_OF_FILE = 0xC0000011
 
   # The server's response from when we {Tree#create created} this File
   #
   # @return [Smb2::Packet::CreateResponse]
   attr_accessor :create_response
 
+  # @return [String]
```

Comment:
This should have a description even if most people will understand want a filename is.  Explain what should go in Smb2::Tree#share vs Smb2::File#filename by showing an example full UNC path being broken up into each piece.